Felienne/hedy
# [UI FIX] Improves styling of landing page elements

**Description**
In this PR we improve the styling of the landing page elements. We also make a very small start on the issue of improving the overall responsiveness of the site as this is lacking in a lot of spots (#2776).

**Fixes**
This PR fixes #2934.

**How to test**
Make sure you are logged in and navigate to `/landing-page`. Verify that the options always look nice.

![Schermafbeelding 2022-06-21 om 15 19 05](https://user-images.githubusercontent.com/38155424/174809498-312fd985-ffcf-4884-b22a-99dc902b3df1.png)
![Schermafbeelding 2022-06-21 om 15 18 57](https://user-images.githubusercontent.com/38155424/174809493-2a7f7ad1-c48c-4269-ada4-080541cb7e8c.png)
